CRM/Campaign/Form/Task/Interview.php

```diff
@@ -580,10 +581,10 @@ public function getVoterIds() {
     $this->_contactIds = $this->get('contactIds');
     if (!is_array($this->_contactIds)) {
       //get the survey activities.
-      $activityStatus = CRM_Core_PseudoConstant::activityStatus('name');
+      $scheduledStatusId = CRM_Core_PseudoConstant::getKey('CRM_Activity_DAO_Activity', 'activity_status_id', 'Scheduled');
       $statusIds = array();
-      if ($statusId = array_search('Scheduled', $activityStatus)) {
-        $statusIds[] = $statusId;
+      if ($scheduledStatusId) {
```

Comment:
should this not be if ($scheduledStatusId === $statusId) {

Reply:
@eileenmcnaughton No, the original code was checking if $scheduledStatusId had a value so I just updated it.  However, looking again, there is scope for simplifying the code a little which removes that `if` entirely.  Does it look ok now?